app/schema.graphql

```diff
@@ -48,6 +48,9 @@ type Dimension implements Node {
 
   """The data type of the column. Categorical or numeric."""
   dataType: DimensionDataType!
+
+  """Whether the dimension data is continuous or discrete."""
+  shape: DimensionShape!
```

Comment:
@RogerHYang I know shape is kinda a conflated term wrt numpy but this was my best attempt at an abstract noun that made sense.

Reply:
shape makes sense to me, and i also like it. at any rate, we can also punt this to a future wordsmith

Reply:
another suggestion could be `DimensionDomain` ([source](https://www.collegeprepalgebra.com/cpa/wp-content/uploads/additional_lessons/discrete_and_continuous_domains.pdf)), or if we want to be more down-to-earth, `DimensionLumpiness` haha